docker-compose-prod.yaml

```diff
@@ -0,0 +1,13 @@
+version: '3.9'
+
+services:
+  zkitterd:
+    build:
+      context: .
+      dockerfile: Dockerfile-prod
+    ports:
+      - '3000:3000'
+
+secrets:
+  config:
+    file: config.prod.json
```

Comment:
@0xtsukino could you check how our hosting provider manage secrets for docker images?
Usually they have a CLI or GUI to set secrets.
Depending on wether they are managed as env variables or files, the logic to consume them in the app will be slightly different.